node_modules/gh-users/lib/api.js

```diff
@@ -784,3 +789,106 @@ var _getCalendarInfo = function(ctx, user, format) {
     var link = util.format('https://%s/api/user/%s/%s/calendar.%s', ctx.app.host, user.id, user.calendarToken, format);
     return new CalendarInfo(ctx.app, user.displayName, '', ctx.app.displayName, link, '', user.updatedAt);
 };
+
+/* Terms and conditions */
+
+/**
+ * Get the Terms and Conditions for an application
+ *
+ * @param  {Number}    appId        The application for which to retrieve the terms and conditions
+ * @return {Object}                 The Terms and Conditions for the application
+ */
+var getTermsAndConditions = module.exports.getTermsAndConditions = function(appId) {
+    return {
+        'text': ConfigAPI.config(appId).termsAndConditionsText,
+        'enabled': ConfigAPI.config(appId).termsAndConditionsEnabled,
+        'lastUpdate': ConfigAPI.config(appId).termsAndConditionsLastUpdate
+    };
+};
```

Comment:
Is the user api really the best spot for this, as opposed to `gh-app` or `gh-config` ?